### PR TITLE
[new] Addd MST as a Data Source in Google Data Studio

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

Validated and updated the article related to adding MST as a Data Source.
This [article](https://app.intercom.com/a/apps/uvcadzno/articles/articles/3235774/show) was outdated. I did not have to use openssl to add MST as a Data Source.

Also fixes a couple of typos.

# Links

Fixes https://github.com/timescale/docs/issues/1596

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

**Review by Loquacity — APPROVED** [review]

File: mst/google-data-studio-mst.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: Integrate Managed Service for TimescaleDB and Google Data Studio
+excerpt: Integrate Google Data Studiol for fully managed services on AWS, Azure, or GCP.
+product: mst
+---
+# Integrate Managed Service for TimescaleDB and Google Data Studio
```
Comment (charislam) [inline mst/google-data-studio-mst.md:6]:
```suggestion
---

# Integrate Managed Service for TimescaleDB and Google Data Studio
```

File: mst/google-data-studio-mst.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+title: Integrate Managed Service for TimescaleDB and Google Data Studio
+excerpt: Integrate Google Data Studiol for fully managed services on AWS, Azure, or GCP.
```
Comment (charislam) [inline mst/google-data-studio-mst.md:3]:
```suggestion
excerpt: Integrate Google Data Studio for fully managed services on AWS, Azure, or GCP.
```

File: mst/google-data-studio-mst.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+product: mst
+---
+# Integrate Managed Service for TimescaleDB and Google Data Studio
+
+You can create reports or perform some analysis on data you have in Managed
+Service for TimescaleDB using Google data studio. You can use Data studio to
```
Comment (charislam) [inline mst/google-data-studio-mst.md:9]:
```suggestion
Service for TimescaleDB using Google Data Studio. You can use Data Studio to
```

File: mst/google-data-studio-mst.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+integrate other data sources such as, YouTube Analytics, MySQL, BigQuery,
```
Comment (charislam) [inline mst/google-data-studio-mst.md:10]:
```suggestion
integrate other data sources, such as YouTube Analytics, MySQL, BigQuery,
```

File: mst/google-data-studio-mst.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+AdWords, and others.
+
+## Before you begin
+
+*   You should also have a Google account.
+*   In the service overview page of the Managed Service for TimescaleDB service:
+    *   Download he CA certificate anmed `ca.pem` for your Managed Service for
```
Comment (charislam) [inline mst/google-data-studio-mst.md:17]:
```suggestion
    *   Download the CA certificate named `ca.pem` for your Managed Service for
```

File: mst/google-data-studio-mst.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+        TimescaleDB service.
+    *   Make a note of the `Host`, `Port`, `Database name`, `User`, and `Password`
+        fields for the Managed Service for TimescaleDB service.
+
+<Procedure>
```
Comment (charislam) [inline mst/google-data-studio-mst.md:22]:
```suggestion
<procedure>
```

File: mst/google-data-studio-mst.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+### Connecting to Managed Service for TimescaleDB datasource from Data Studio
```
Comment (charislam) [inline mst/google-data-studio-mst.md:24]:
```suggestion
### Connecting to Managed Service for TimescaleDB data source from Data Studio
```

File: mst/google-data-studio-mst.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+1.  Login to Google and open [Google Data Studio][google-data-studio].
```
Comment (charislam) [inline mst/google-data-studio-mst.md:26]:
```suggestion
1.  Log in to Google and open [Google Data Studio][google-data-studio].
```

File: mst/google-data-studio-mst.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+1.  Click the `Create` + button and choose `Data source`.
```
Comment (charislam) [inline mst/google-data-studio-mst.md:27]:
```suggestion
1.  Click the `Create +` button and choose `Data source`.
```

File: mst/google-data-studio-mst.md [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+1.  Select the `PostgreSQL` as Google Connector.
```
Comment (charislam) [inline mst/google-data-studio-mst.md:28]:
```suggestion
1.  Select `PostgreSQL` as the Google Connector.
```

File: mst/google-data-studio-mst.md [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+1.  In the `Database Authentication` tab, type details for the `Host Name`,
+    `Port`, `Database`, `Username`, and `Password` fields.
+1.  Select `Enable SSL` and upload your server certificate file, `ca.pem`.
+1.  Click `AUTHENTICATE`.
+1.  Choose the table to be queried, or select `CUSTOM QUERY` to create an SQL query.
+1.  Click `CONNECT`.
+
+</Procedure>
```
Comment (charislam) [inline mst/google-data-studio-mst.md:36]:
```suggestion
</procedure>
```

**Review by charislam — COMMENTED** [review]
Needs a page-index entry

**Comment by Rajakavitha1** [comment]
> Needs a page-index entry

Yes!!! Thanks @charislam index entry coming in the next PR.

**Pull request merged**